api-gateway/src/main/java/com/petclinic/bffapigateway/dtos/Cart/CartProductResponseDTO.java

```diff
@@ -17,4 +17,4 @@ public class CartProductResponseDTO {
     private Double productSalePrice;
     private Double averageRating;
     private Integer quantityInCart;
-}
+    private Integer productQuantity;}
```

Comment:
what is the difference here between productQuantity and quantityInCart?

Reply:
`quantityInCart `is used to track how many units are currently in the cart. I didn’t want to change or reuse it since it might affect other functionality, so I added `productQuantity `to represent the available stock separately. This keeps things clear and prevents any conflicts with existing code.

Reply:
thank you for the clarification.